tests/test_mixed.py

```diff
@@ -0,0 +1,153 @@
+B_hat_prim = U_theta_phi_alpha(theta=-theta_A + pi, phi=-alpha_A, alpha=-phi_A + pi / 2)
+
+gamma_A, gamma_B = sp.symbols('gamma_A gamma_B', real=True)
+
+Expr = Any
+
+
+@pytest.fixture
+def alice() -> MixedStrategy:
+    return MixedStrategy([
+        (sp.cos(gamma_A / 2) ** 2, A_hat),
+        (sp.sin(gamma_A / 2) ** 2, A_hat_prim),
+    ])
+
+
+@pytest.fixture
+def bob() -> MixedStrategy:
+    return MixedStrategy([
+        (sp.cos(gamma_B / 2) ** 2, B_hat),
+        (sp.sin(gamma_B / 2) ** 2, B_hat_prim),
+    ])
+
+
+@pytest.fixture
+def mixed_ewl(alice: MixedStrategy, bob: MixedStrategy) -> MixedEWL:
+    return MixedEWL(psi=psi, C=C, D=D, players=[alice, bob], payoff_matrix=payoff_matrix)
+
+
+@pytest.fixture
+def mixed_ewl_fixed(mixed_ewl: MixedEWL) -> MixedEWL:
+    return mixed_ewl.fix(theta_A=pi / 2, phi_A=0, alpha_A=0)
+
+
+def test_kraus() -> None:
+    assert D == kraus(C, D)
+    assert C == kraus(D, C)
+
+    M = sp.Matrix([[0, 1], [1, 0]])
+    assert M == kraus(D, M)
+
+
+def test_MixedStrategy_invalid_sum() -> None:
+    with pytest.raises(AssertionError):
+        MixedStrategy([(0.9, C), (0.2, D)], check_sum=True)
+
+
+def test_MixedStrategy_params(alice: MixedStrategy, bob: MixedStrategy) -> None:
+    assert alice.params == {gamma_A, theta_A, alpha_A, phi_A}
+    assert bob.params == {gamma_B, theta_A, alpha_A, phi_A}
+
+
+def test_MixedStrategy_fix(alice: MixedStrategy) -> None:
+    assert not alice.fix(theta_A=pi / 2, phi_A=0, alpha_A=0, gamma_A=sp.sympify('1/2')).params
+
+
+def test_MixedEWL_number_of_players(mixed_ewl: MixedEWL) -> None:
+    assert mixed_ewl.number_of_players == 2
+
+
+def test_MixedEWL_params(mixed_ewl: MixedEWL, mixed_ewl_fixed: MixedEWL) -> None:
+    assert mixed_ewl.params == {gamma_A, gamma_B, theta_A, alpha_A, phi_A}
+    assert mixed_ewl_fixed.params == {gamma_A, gamma_B}
+
+
+def test_MixedEWL_fix(mixed_ewl: MixedEWL, mixed_ewl_fixed: MixedEWL) -> None:
+    theta_A_fix, phi_A_fix, alpha_A_fix = pi / 2, 0, 0
+
+    A_hat_fix = U_theta_phi_alpha(theta=theta_A_fix, phi=phi_A_fix, alpha=alpha_A_fix)
+    B_hat_fix = U_theta_phi_alpha(theta=theta_A_fix + pi, phi=alpha_A_fix, alpha=phi_A_fix - pi / 2)
+    A_hat_prim_fix = U_theta_phi_alpha(theta=theta_A_fix, phi=phi_A_fix - pi / 2, alpha=alpha_A_fix - pi / 2)
+    B_hat_prim_fix = U_theta_phi_alpha(theta=theta_A_fix + pi, phi=alpha_A_fix - pi / 2, alpha=phi_A_fix - pi)
+
+    gamma_A, gamma_B = sp.symbols('gamma_A gamma_B', real=True)
+
+    alice_fixed: MixedStrategy = MixedStrategy([
+        (sp.cos(gamma_A / 2) ** 2, A_hat_fix),
+        (sp.sin(gamma_A / 2) ** 2, A_hat_prim_fix),
+    ])
+
+    bob_fixed: MixedStrategy = MixedStrategy([
+        (sp.cos(gamma_B / 2) ** 2, B_hat_fix),
+        (sp.sin(gamma_B / 2) ** 2, B_hat_prim_fix),
+    ])
+
+    manually_fixed_ewl = MixedEWL(psi=psi, C=C, D=D, players=[alice_fixed, bob_fixed], payoff_matrix=payoff_matrix)
```

Comment:
I don't think we need to create `manually_fixed_ewl` here, let's just compare with `alice_fixed` and `bob_fixed` in the asserts below